examples/00-mapdl-examples/bracket_static.py

```diff
@@ -0,0 +1,743 @@
+#    which will pop up on screen.
+#
+
+###############################################################################
+# Create first circle
+# ~~~~~~~~~~~~~~~~~~~
+#
+# With the use of logic/booleans geometrical operations, the original geometric
+# parameters (``box1``, ``box2``) can be used to locate the circles.
+#
+# Create the half circle at each end of the bracket. You will actually create
+# a full circle on each end and then combine the circles and rectangles with a
+# Boolean add operation (discussed in `Subtract pin holes from bracket`_).
+#
+# The APDL command to create the circles is
+# :meth:`mapdl.cyl4() <ansys.mapdl.core.Mapdl.cyl4>`.
+#
+# The first circle area is located on the left side at the X,Y location and
+# its radius is :math:`1`.
+
+# Create the first circle
+radius = 1
+circle1_X = box1[0]
+circle1_Y = (box1[2] + box1[3]) / 2
+mapdl.cyl4(circle1_X, circle1_Y, radius)
+
+mapdl.aplot(vtk=True, cpos="xy", show_area_numbering=True, show_lines=True)
+
+
+###############################################################################
+# Create second circle
+# ~~~~~~~~~~~~~~~~~~~~
+#
+# The second circle to be created is at the X,Y location:
+#
+
+circle2_X = (box2[0] + box2[1]) / 2
+circle2_Y = box2[3]
+
+###############################################################################
+# Use these parameter values to create the new area with the same radius of :math:`1`
+# as the first circle area.
+
+mapdl.cyl4(circle2_X, circle2_Y, radius)
+mapdl.aplot(vtk=True, cpos="xy", show_area_numbering=True, show_lines=True)
+
+
+###############################################################################
+# Add areas
+# ~~~~~~~~~
+# Now that the appropriate pieces of the model (rectangles and circles) are defined,
+# add them together so the model becomes one continuous area.
+# Use the Boolean add operation :meth:`mapdl.aadd() <ansys.mapdl.core.Mapdl.aadd>`
+# to add the areas together.
+#
+# Pick All for all areas to be added.
+mapdl.aadd("all")  # Prints the id of the created area
+
+###############################################################################
+# Create line fillet
+# ~~~~~~~~~~~~~~~~~~
+#
+# The right angle between the two boxes will be improved using a fillet with a
+# radius of :math:`0.4`.
+# This can be done by selecting the lines around that area and creating a fillet.
+#
+# The APDL :meth:`mapdl.lsel() <ansys.mapdl.core.Mapdl.lsel>` command is used
+# to select lines. Here we use the X and Y location of the lines used to create
+# the boxes to create our selection.
+#
+# After we select the line it needs to be written to a parameter so we can use
+# it to generate the fillet line.
+# That is done using the :meth:`mapdl.get() <ansys.mapdl.core.Mapdl.get>` command
+# from PyMAPDL or `GET` in pymapdl.
+#
+# Since we have selected one line, we can use the ``MAX`` and ``NUM`` arguments
+# for the :meth:`mapdl.get() <ansys.mapdl.core.Mapdl.get>` command.
```

Comment:
```suggestion
# Because you have selected one line, you can use the ``MAX`` and ``NUM`` arguments
# for the :meth:`mapdl.get() <ansys.mapdl.core.Mapdl.get>` method.
```